packages/salesforcedx-vscode-core/src/notifications/notificationService.ts

```diff
@@ -12,6 +12,7 @@ import { DEFAULT_SFDX_CHANNEL } from '../channels';
 import { STATUS_BAR_MSG_TIMEOUT_MS } from '../constants';
 import { nls } from '../messages';
 import { sfdxCoreSettings } from '../settings';
+// import { stringify } from 'querystring';
```

Comment:
let's remove this import if it's not being used

Reply:
Sounds good. I wasn't sure should I keep it or not but sure I will update it.